runtime/src/main/java/org/corfudb/runtime/view/replication/QuorumReplicationProtocol.java

```diff
@@ -98,6 +104,13 @@ public ILogData peek(RuntimeLayout runtimeLayout, long address) {
         }
     }
 
+    @Override
+    public Map<Long, ILogData> readRange(RuntimeLayout runtimeLayout, Set<Long> globalAddresses) {
+        return globalAddresses.parallelStream()
```

Comment:
why parallel stream?

Reply:
Just pushing code from the abstract class to the  concrete implementation. This used to be the abstract class' default implementation.